src/protobuf/WireFormats.proto

```diff
@@ -82,7 +82,7 @@ message AkkaControlMessage {
 message AkkaHandshakeInfo {
 	AddressData origin = 1;
 	fixed64 uid = 2;
-	string cookie = 3;
+  optional string cookie = 3;
```

Comment:
I don't think this is needed - these fields are already optional by default.

Reply:
Everything in Proto3 is optional... I'll undo this.

Reply:
No worries - I took care of it